logger.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// VerbosityLevel maps to a level of logs to be enabled on the command
+type VerbosityLevel uint8
+
+const (
+	// VerbosityLevelOff no log messages displayed
+	VerbosityLevelOff VerbosityLevel = iota
+	// VerbosityLevelInfo enables printing up to Info level command log statements
+	VerbosityLevelInfo
+	// VerbosityLevelDebug enables printing up to Debug level command log statements
+	VerbosityLevelDebug
+)
+
+// getLoggerVerbosity takes a VerbosityLevel and returns an appropriate logging level
+func getLoggerVerbosity(lvl VerbosityLevel) zapcore.Level {
+	zlvl := zap.DebugLevel
+	switch lvl {
+	case VerbosityLevelOff:
+		zlvl = zap.WarnLevel
+	case VerbosityLevelInfo:
+		zlvl = zap.InfoLevel
+	}
+	return zlvl
+}
+
+func configureLoggerConfig(opts *Options) zap.Config {
+	loggerConfig := zap.NewDevelopmentConfig()
+	loggerConfig.EncoderConfig = zapcore.EncoderConfig{
```

Comment:
this matches what the default development config is already, not sure we need to set them again